src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size-cell;
+  max-width: 7 * $width-cell + 6 * $gap-size-cell;
```

Comment:
If you know hot to use css function - it would be great to use this calculation there. Bit it's not required